apache/zeppelin
# [ZEPPELIN-1229] Clear browser cache with version hashtag.

### What is this PR for?
This PR avoids browser cache.

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1229

### How should this be tested?
1. Run old version of zeppelin(I was using 0.6.1) and show main page up in your browser.
2. Stop zeppelin.
3. Run this PR of Zeppelin and show page up on the same browser of 1.
4. check main page changed.

- before (Job menu doesn't show)
![2017-01-06 01_50_29](https://cloud.githubusercontent.com/assets/3348133/21728478/43aec6d0-d3fb-11e6-84ce-9e1b18853611.gif)

- after (Job menu show)
![2017-01-06 01_55_10](https://cloud.githubusercontent.com/assets/3348133/21728482/4a794e72-d3fb-11e6-8934-d6f92d65889b.gif)


### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
